tooling/metamodel-generator/src/main/java/org/hibernate/processor/util/xml/XmlParserHelper.java

```diff
@@ -120,7 +121,10 @@ public <T> T getJaxbRoot(InputStream stream, Class<T> clazz, Schema schema)
 		ContextProvidingValidationEventHandler handler = new ContextProvidingValidationEventHandler();
 		try {
 			staxEventReader = new JpaNamespaceTransformingEventReader( staxEventReader );
-			JAXBContext jaxbContext = JAXBContext.newInstance( ObjectFactory.class );
+			ClassLoader cl = this.getClass().getClassLoader();
+			String packageName = Optional.ofNullable(ObjectFactory.class.getPackage())
```

Comment:
How and what and under what conditions can `ObjectFactory.class.getPackage()` return `null`?

According to the Javadoc, `Class.getPackage()` only returns null when ... _this class represents an array type, a primitive type or void_.

Reply:
Yes I know, it won't but I get "stupid" error from code analysis module in gradle build 😄